lib/response.js

```diff
@@ -368,6 +371,10 @@ res.sendFile = function sendFile(path, options, callback) {
     done = options;
     opts = {};
   }
+  
+  if (app.settings.etag == false) {
+        opts.etag = app.settings.etag;
```

Comment:
I would expect that if the user explicitly set the `etag` option to `res.sendFile`, it would not be overwritten by the setting.